cumulusci/tasks/github.py

```diff
@@ -207,3 +210,51 @@ def _run_task(self):
                 )
 
                 self.logger.info('Merge conflict on branch {}: created pull request #{}'.format(branch.name, pull.number))
+
+
+class CommitApexDocs(BaseGithubTask):
+
+    task_options = {
+        'branch': {
+            'description': 'branch name',
+            'required': True,
+        },
+        'dir_local': {
+            'description': 'local dir of ApexDocs (contains index.html)',
+            'required': True,
+        },
+        'dir_repo': {
```

Comment:
Could certainly be master branch if a user keeps their docs in master, or could otherwise share history with master. And remote doesn't really imply any branch since it is independent of branches in a way. Let's go with `dir_remote` if that's okay with you.